moveit_ros/planning/planning_request_adapter_plugins/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+add_compile_options(-Wno-unused-parameter)
```

Comment:
Please move this into the top-level CMakeLists.txt and add a TODO to remove this. After the sync all these warnings should be resolved anyway.

Reply:
Done. Moved it to the top-level CMakeLists.txt. Please check it [here](https://github.com/ros-planning/moveit2/pull/114/files#diff-8a8b87aaf16323bb476afbe7bfd14fbdR9-R15).